atlite/pv/irradiation.py

```diff
@@ -186,6 +187,7 @@ def clip(influx, influx_max):
         diffuse_t = (1.0 + cos_surface_slope) / 2.0 * diffuse + _albedo(
             ds, influx
         ) * influx * ((1.0 - cos_surface_slope) / 2.0)
+        ground_t = None
```

Comment:
I believe this is wrong.

1. This will probably lead to errors with the `.convert_and_aggregate(...)` method, as the type is not an `xr.DataArray`.
2. The ground reflected irradiation is contained in the `diffuse_t` calculation above; c.f. with second term containing `_albedo`.

--> separate the calculation of `diffuse_t` and `ground_t` for `trigon_model == "simple"` and return the correct `ground_t`.

Reply:
Ground reflected irradiation now separated from `diffuse_t` for the `simple` `trigon_model`. I hope I did it right, as I'm by no means an expert in solar irradiation calculations.